javav2/example_code/cloudfront/Readme.md

```diff
@@ -4,67 +4,173 @@ This README discusses how to run and test the Java code examples for Amazon Clou
+| DeleteDistributionResources |                                    | Deletes the origin access control, key group, and public key.                                                                                                                                                                                                                                                                                     |
+
+`CloudFrontSigningTest` programmatically creates a CloudFront distribution along with the required setup 
+to test that signed URLs/cookies are required to access S3 items through CloudFront. As part of the setup, 
+the `CloudFrontSigningTest` prepares and deletes an S3 bucket along with uploading and deleting 
+the provided file, `index.html`. All resources created during the run of the tests are deleted at the end.
+
+The `CloudFrontSigningTest` tests take several minutes to run. Deploying the distribution, so that the test can run for signing URLs and cookies,
+takes a couple of minutes. Undeploying the distribution, so that it can be deleted, also takes a couple of minutes.
+
+You can execute the JUnit tests from a Java IDE, such as IntelliJ, or from the command line by using Maven. As each test
+runs, messages are logged that inform you of what the tests are doing and if a test succeeds or fails. For example, the following message
+informs you that Test 3 passed.
 
 	Test 3 passed
 
 **WARNING**: _Running these JUnit tests manipulates real Amazon resources and may incur charges on your account._
 
- ### Properties file
-Before running the CloudFront JUnit tests, you must define values in the **config.properties** file located in the **resources** folder. This file contains values that are required to execute the JUnit tests. For example, you define an application name. If you do not define all values, the JUnit tests fail.
+### Properties file
 
-Define these values to successfully run the JUnit tests:
+Before running the CloudFront JUnit tests, you must define values in the **config.properties** file located in the
+`src/main/resources` folder. This file contains values that are required to execute the JUnit tests. 
+If you do not define all values, the JUnit tests fail.
 
-- **functionName** - the name of the function to create.  
-- **filePath** – the path to a file that contains the application logic for the function.
-- **ifMatchVal** - The current version (ETag value) of the function that you are deleting, which you can get using DescribeFunction. 
+Define the following values to run the JUnit tests:
 
+**distributionId** - The distribution id of an exisiting CloudFront distribution. Follow [these steps](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/GettingStarted.SimpleDistribution.html) to manually
+set up a simple CloudFront distribution using the AWS console if you don't already have a distribution. 
+Use the distribution id as the value of this property (something similar to `E1LK64FWG38EAW`)
```

Comment:
**distributionId** - The distribution id of an existing CloudFront distribution. If you don't have a distribution yet, set one up in the console by following [these steps](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/GettingStarted.SimpleDistribution.html). 
Use the distribution Id as the value of this property (for example, `E1LK64FWG38EAW`).